### ISSUE TITLE
Empty collations (Source PDFs are available)

### ISSUE THREAD

**Issue by VermaBSI** [ben-garside/XCP#73]
CLAIMED list In the Collations tab in PROD, 25/09/2015, 16.15 hrs:

All items still at 10.10 - XCP has collated nothing for these (i.e. no PDF either: error message is in the zips). I was able to obtain the PDFs from CoLos. If necessary please could these be removed to be discovered through the collation operation again.

**Comment by ben-garside** [comment]
There are now only the following items that have got missing collated files:

 - XCP3089229
 - XCP9453057
 - XCP3684731
 - XCP4685993
 - XCP5951046
 - XCP4049451
 - XCP8867091
 - XCP5630554
 - XCP6014416
 - XCP7086541

I am looking in to these

**Comment by VermaBSI** [comment]
XCP7078844_30159218_p8.zip
XCP8879969_30159218_p2.zip

**Comment by VermaBSI** [comment]
XCP7078844_30159218_p8.zip
XCP8879969_30159218_p2.zip

XCP9788270_30297353_p2.zip
XCP6028475_30297353_p8.zip

XCP5119268_30298804_p2.zip
XCP7299034_30298804_p8.zip

XCP5359153_30281561_p8.zip

XCP4280101_30284311_p8.zip
XCP9476832_30284311_p4.zip

XCP0620126_30261091_p8.zip

XCP3613093_30331419_p1.zip

XCP0476928_30276374_p8.zip

XCP0030515_30270463_p4.zip
XCP6523055_30270463_p8.zip

XCP1917968_30329879_p8.zip

XCP0032619_30282335_p8.zip

XCP5630554_30068604_p8.zip

XCP8867091_30068495_p8.zip

XCP4049451_30065800_p8.zip

XCP4685993_30061582_p8.zip

XCP3684731_30009185_p8.zip

XCP5951046_30061587_p8.zip

**Comment by VermaBSI** [comment]
Hi Ben,

All these recent collations listed above in XCP PROD have not found any PDF source file so cannot be sent.

I have organised them in UPI pairs - the majority listed above NOT listed as a pair is because the sister pipeline item is missing as well. 

I will add those to issue #53 too.

**Comment by VermaBSI** [comment]
To add - all items listed above are in the CLAIMED list in the Activity Tracker Collation tab. Have left them at 10.10.

**Comment by ben-garside** [comment]
Most of these have no been located, the rest we genuinely do not have files for:

 - XCP9453057
 - XCP3684731
 - XCP4685993
 - XCP5951046
 - XCP4049451
 - XCP8867091
 - XCP5630554
 - XCP6014416
 - XCP7086541

**Comment by VermaBSI** [comment]
Yeh I am just seeing a whole load of items with content in them now, cool!...I will set these at 10.20 eventually and then let you know - and I will then leave it up to you whether you want to run any sending today or Monday depending on how busy you are :)

**Comment by ben-garside** [comment]
I have made another change to the logic for this. The main issue here is that when an item is being allocated a pipeline it does the check for the core XML and if it doesn't find it it just defaults to assigning it to a pl2. The process will now confirm this by doing a look up in to COLOS to see if the PDF exists or not, if it doesn't it will assign an error code to the pipeline so we know there is an issue. Until the content is then available in the future the system will not try to collate the files.

**Comment by VermaBSI** [comment]
Makes sense - yes it should check for the PDF for pl 2 as well.  I guess we've always assumed PDFs would always be available as a default for any Material reached PUBL.

Have set all I want to 10.20 for the send on Monday :)

**Comment by ben-garside** [comment]
Yeah previously it was just an assumption, bit now we will actually do a check. The benefit of this is that if the content should become available again in the future it will automatically be collated and set to 10:10
